pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
+asyncio_mode = auto
```

Comment:
I haven't found other projects that have a pytest.ini file. Maybe there's a better way to suppress the warnings.